src/plots/gl2d/camera.js

```diff
@@ -262,34 +262,27 @@ function createCamera(scene) {
         var lastX = result.lastPos[0],
             lastY = result.lastPos[1];
 
-        switch(scene.fullLayout.dragmode) {
-            case 'zoom':
-                break;
-
-            case 'pan':
-                var scale = Math.exp(0.1 * dy / (viewBox[3] - viewBox[1]));
-
-                var cx = lastX /
-                        (viewBox[2] - viewBox[0]) * (dataBox[2] - dataBox[0]) +
-                    dataBox[0];
-                var cy = lastY /
-                        (viewBox[3] - viewBox[1]) * (dataBox[3] - dataBox[1]) +
-                    dataBox[1];
-
-                dataBox[0] = (dataBox[0] - cx) * scale + cx;
-                dataBox[2] = (dataBox[2] - cx) * scale + cx;
-                dataBox[1] = (dataBox[1] - cy) * scale + cy;
-                dataBox[3] = (dataBox[3] - cy) * scale + cy;
-
-                scene.setRanges(dataBox);
-
-                result.lastInputTime = Date.now();
-                unSetAutoRange();
-                scene.cameraChanged();
-                scene.handleAnnotations();
-                scene.relayoutCallback();
-                break;
-        }
+        var scale = Math.exp(scene.fullLayout.zoomspeed * 10.0 * dy / (viewBox[3] - viewBox[1]));
```

Comment:
Nice. Can we put that `10.0` in https://github.com/plotly/plotly.js/blob/master/src/constants/interactions.js ?